src/Aspire.Hosting.Testing/ResourceLoggerForwarderService.cs

```diff
@@ -10,7 +10,7 @@ namespace Aspire.Hosting.Testing;
 /// <summary>
 /// A background service that watches resource logs and forwards them to the host's <see cref="ILogger"/> infrastructure.
 /// </summary>
-internal sealed class ResourceLoggerForwarderService(
+public sealed class ResourceLoggerForwarderService(
```

Comment:
@DamianEdwards @ReubenBond @mitchdenny - thoughts on making this public in `Aspire.Hosting.Testing`?

Reply:
To be honest I'd rather not make this API public at this point in time. I feel like if we were going to provide a mechanism to do this we'd probably have some kind of discrete interface that people could resolve to subscribe to log events or even just register a Logger<T> to forward the logs to.

I probably wouldn't expose the actual logging pump (the background service).

Reply:
Can someone make an API proposal? This seems like a common scenario - we are using it in a bunch of hosting resource tests now.